plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -541,9 +541,9 @@ codemodernizer.chat.message.button.view_summary=View summary
 codemodernizer.chat.message.error_request=Request failed
 codemodernizer.chat.message.follow_up.new_transformation=Start a new transformation
 codemodernizer.chat.message.local_build_begin=I'm building your module. This can take up to 10 minutes, depending on the size of your module.
-codemodernizer.chat.message.local_build_failed=Sorry, I couldn't run the Maven clean install command to build your module.
```

Comment:
Is there no way to escape the single quote character?

Reply:
Have we asked the toolkit team about this. Usually for JS/TS you have string escape delimiters for special characters.


-    \\ – Backslash
 -   \' – Single quote
   -  \" – Double quote
    - \n – Newline
-    \r – Carriage return
  -  \t – Tab

Reply:
Have we asked the toolkit team about this. Usually for JS/TS you have string escape delimiters for special characters.


-    \\ – Backslash
-   \' – Single quote
   -  \" – Double quote
    - \n – Newline
-    \r – Carriage return
  -  \t – Tab

Reply:
https://stackoverflow.com/questions/4449639/apostrophe-doesnt-get-translated-properly-when-placed-in-a-resource-bundle

Reply:
Spoke to PM and we don't want contractions in our messages anyway. So `could not` is preferred over `couldn't`.